autoscale_cloudroast/test_repo/autoscale/system/schedule_policies/test_system_scheduler_update_group.py

```diff
@@ -143,23 +145,23 @@ def test_system_upd_launch_config_cron_style_scheduler(self):
             sp_cooldown=3600,
             sp_change=self.sp_change,
             schedule_cron='* * * * *')
-        sleep(60 + self.scheduler_interval)
         active_servers = self.sp_change + group.groupConfiguration.minEntities
         active_after_scaleup = self.wait_for_expected_number_of_active_servers(
-            group_id=group.id,
-            expected_servers=active_servers)
-        upd_lc_server = set(
-            active_after_scaleup) - set(active_list_b4_upd)
+            group_id=group.id, expected_servers=active_servers,
+            timeout=(60 + self.scheduler_interval), interval_time=2,
```

Comment:
Yeah. You are right. There should be a few seconds margin for safety. I'll change it in all the tests. Thanks :)